## 71% of end-end tests have passed
```
Total executed tests: 7
Failed tests: 2
Expected to fail tests: 0
Passed tests: 5
```
```
IDs of failed tests: 727230,727229 
```
<h3>Failed tests (2)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/syncing-state&PR_ID=21298&APK_NAME=21298.apk&TR_CASE_IDS=727230,727229">Rerun failed tests</a></li><br/><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3687521">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td>

```
critical/test_wallet.py:163: in test_wallet_send_asset_from_drawer
    self.wallet_1.navigate_back_to_wallet_view()
../views/base_view.py:455: in navigate_back_to_wallet_view
    while not element.is_element_displayed(1) and counter <= attempts:
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 A session is either terminated or not started
E   Stacktrace:
E   NoSuchDriverError: A session is either terminated or not started
E       at asyncHandler (/mnt/sauce/appium/appium-v2.11.0/packages/base-driver/lib/protocol/protocol.js:309:15)
E       at /mnt/sauce/appium/appium-v2.11.0/packages/base-driver/lib/protocol/protocol.js:512:15
E       at Layer.handle [as handle_request] (/mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/layer.js:95:5)
E       at next (/mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/route.js:149:13)
E       at Route.dispatch (/mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/route.js:119:3)
E       at Layer.handle [as handle_request] (/mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/layer.js:95:5)
E       at /mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/index.js:284:15
E       at param (/mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/index.js:365:14)
E       at param (/mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/index.js:376:14)
E       at Function.process_params (/mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/index.js:421:3)
E       at next (/mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/index.js:280:10)
E       at logger (/mnt/sauce/appium/appium-v2.11.0/node_modules/morgan/index.js:144:5)
E       at Layer.handle [as handle_request] (/mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/layer.js:95:5)
E       at trim_prefix (/mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/index.js:328:13)
E       at /mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/index.js:286:9
E       at Function.process_params (/mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/index.js:346:12)
E       at next (/mnt/sauce/appium/appium-v2.11.0/node_modules/express/lib/router/index.js:280:10)
E       at /mnt/sauce/appium/appium-v2.11.0/node_modules/body-parser/lib/read.js:137:5
E       at AsyncResource.runInAsyncScope (node:async_hooks:204:9)
E       at invokeCallback (/mnt/sauce/appium/appium-v2.11.0/node_modules/raw-body/index.js:238:16)
E       at done (/mnt/sauce/appium/appium-v2.11.0/node_modules/raw-body/index.js:227:7)
E       at IncomingMessage.onEnd (/mnt/sauce/appium/appium-v2.11.0/node_modules/raw-body/index.js:287:7)
E       at IncomingMessage.emit (node:events:513:28)
E       at endReadableNT (node:internal/streams/readable:1359:12)
E       at processTicksAndRejections (node:internal/process/task_queues:82:21)
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3687520">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Expected amount of confirmations is 7, in fact 9604</div><div>Expected amount of confirmations is 7, in fact 9678</div></blockquote></p>

```
critical/test_wallet.py:142: in test_wallet_send_eth
    self.network_api.wait_for_confirmation_of_transaction(address=self.sender['wallet_address'],
../support/api/network_api.py:131: in wait_for_confirmation_of_transaction
    pytest.fail('The last transaction was not confirmed, address is %s, still has %s confirmations' % (
 The last transaction was not confirmed, address is 0x83DFF07F40642Fb42FEBB5deB6A314c8b725749E, still has 9678 confirmations
```

<br/><br/></td></tr></tbody></table></details><h3>Passed tests (5)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3687517">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d8da12401244677b3d2e98763405d09?auth=d6e3c8bae921e43b25ff29e73ee9a472#100">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e035f4954b8446659593ff4b8d2b4376?auth=488252a718aaad3135e825fb11abae56#91">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3687518">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e788bca953734ce5bf4d2cff65d0d04d?auth=f03db182150a38ff99471118029245c7#168">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1e8801dc9cb24003bcd5b039af715cf6?auth=c5746483a4cb192f3fc9aa2b8c0a1228#112">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3687516">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b59f9a3d7f0c4795b26bd11e30a613fa?auth=7362a1037f6ddf8746fad02fa1895c06#57">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3687519">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b59f9a3d7f0c4795b26bd11e30a613fa?auth=7362a1037f6ddf8746fad02fa1895c06#89">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3687522">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3c7bb7662114458182af16f654058690?auth=58ed20e650d28f1fc946e6c09b63654c#42">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/contexts/onboarding/create_or_sync_profile/view.cljs
+++ b/src/status_im/contexts/onboarding/create_or_sync_profile/view.cljs
@@ -23,6 +23,7 @@
 
 (defn- navigate-to-sign-in-by-syncing
   []
+  (rf/dispatch [:syncing/clear-syncing-fallback-flow])
   (debounce/throttle-and-dispatch
    [:onboarding/navigate-to-sign-in-by-syncing]
    1000))
@@ -37,6 +38,7 @@
 
 (defn- navigate-to-sign-in-by-seed-phrase
   [create-profile?]
+  (rf/dispatch [:syncing/clear-syncing-fallback-flow])
   (rf/dispatch [:onboarding/navigate-to-sign-in-by-seed-phrase
                 (if create-profile?
                   :screen/onboarding.new-to-status

--- a/src/status_im/contexts/onboarding/events.cljs
+++ b/src/status_im/contexts/onboarding/events.cljs
@@ -72,15 +72,21 @@
   [{:keys [db] :as cofx}]
   (let [{:keys [display-name seed-phrase password image-path color] :as profile}
         (:onboarding/profile db)
-        loading-screen (if (seq (:syncing/key-uid db))
+        syncing-account-recovered? (and (seq (:syncing/key-uid db))
+                                        (= (:syncing/key-uid db)
+                                           (get-in db [:onboarding/profile :key-uid])))
+        loading-screen (if syncing-account-recovered?
                          :screen/onboarding.preparing-status
                          :screen/onboarding.generating-keys)]
     (rf/merge cofx
-              {:dispatch       [:navigate-to-within-stack
-                                [loading-screen
-                                 (get db
-                                      :onboarding/navigated-to-enter-seed-phrase-from-screen
-                                      :screen/onboarding.new-to-status)]]
+              {:fx             [[:dispatch
+                                 [:navigate-to-within-stack
+                                  [loading-screen
+                                   (get db
+                                        :onboarding/navigated-to-enter-seed-phrase-from-screen
+                                        :screen/onboarding.new-to-status)]]]
+                                (when-not syncing-account-recovered?
+                                  [:dispatch [:syncing/clear-syncing-installation-id]])]
                :dispatch-later [{:ms       constants/onboarding-generating-keys-animation-duration-ms
                                  :dispatch [:navigate-to-within-stack
                                             [:screen/onboarding.enable-notifications
@@ -154,15 +160,14 @@
         :on-cancel           #(re-frame/dispatch [:pop-to-root :multiaccounts])}}
       {:db (-> db
                (assoc-in [:onboarding/profile :seed-phrase] seed-phrase)
+               (assoc-in [:onboarding/profile :key-uid] key-uid)
                (assoc-in [:onboarding/profile :color] constants/profile-default-color))
        :fx [[:dispatch
              [:navigate-to-within-stack
               [next-screen
                (get db
                     :onboarding/navigated-to-enter-seed-phrase-from-screen
-                    :screen/onboarding.new-to-status)]]]
-            (when-not syncing-account-recovered?
-              [:dispatch [:syncing/clear-syncing-data]])]})))
+                    :screen/onboarding.new-to-status)]]]]})))
 
 (rf/defn navigate-to-create-profile
   {:events [:onboarding/navigate-to-create-profile]}

--- a/src/status_im/contexts/onboarding/syncing/progress/view.cljs
+++ b/src/status_im/contexts/onboarding/syncing/progress/view.cljs
@@ -28,6 +28,7 @@
 
 (defn- navigate-to-enter-seed-phrase
   []
+  (rf/dispatch [:syncing/set-syncing-fallback-flow])
   (debounce/debounce-and-dispatch
    [:onboarding/navigate-to-sign-in-by-seed-phrase :screen/onboarding.sync-or-recover-profile]
    500))

--- a/src/status_im/contexts/profile/login/events.cljs
+++ b/src/status_im/contexts/profile/login/events.cljs
@@ -142,7 +142,7 @@
                            (rf/dispatch [:chats-list/load-success result])
                            (rf/dispatch [:communities/get-user-requests-to-join])
                            (rf/dispatch [:profile.login/get-chats-callback]))}]
-           (when (:syncing/installation-id db)
+           (when (and (:syncing/fallback-flow? db) (:syncing/installation-id db))
              [:dispatch [:pairing/finish-seed-phrase-fallback-syncing]])
            (when-not new-account?
              [:dispatch [:universal-links/process-stored-event]])]})))

--- a/src/status_im/contexts/syncing/events.cljs
+++ b/src/status_im/contexts/syncing/events.cljs
@@ -39,11 +39,6 @@
                     {:type :negative
                      :text error}]))))
 
-(rf/defn initiate-pairing-process
-  {:events [:syncing/initiate-pairing-process]}
-  [{:keys [db]}]
-  {:db (assoc db :syncing/pairing-process-initiated? true)})
-
 (rf/defn set-syncing-installation-id
   {:events [:syncing/set-syncing-installation-id]}
   [{:keys [db]} installation-id key-uid]
@@ -51,15 +46,26 @@
               :syncing/key-uid         key-uid
               :syncing/installation-id installation-id)})
 
-(defn clear-syncing-data
+(defn clear-syncing-installation-id
   [{:keys [db]}]
   {:db (dissoc
         db
         :syncing/key-uid
-        :syncing/installation-id
-        :syncing/pairing-process-initiated?)})
+        :syncing/installation-id)})
 
-(re-frame/reg-event-fx :syncing/clear-syncing-data clear-syncing-data)
+(re-frame/reg-event-fx :syncing/clear-syncing-installation-id clear-syncing-installation-id)
+
+(defn set-syncing-fallback-flow
+  [{:keys [db]}]
+  {:db (assoc db :syncing/fallback-flow? true)})
+
+(re-frame/reg-event-fx :syncing/set-syncing-fallback-flow set-syncing-fallback-flow)
+
+(defn clear-syncing-fallback-flow
+  [{:keys [db]}]
+  {:db (dissoc db :syncing/fallback-flow?)})
+
+(re-frame/reg-event-fx :syncing/clear-syncing-fallback-flow clear-syncing-fallback-flow)
 
 (rf/defn preflight-outbound-check-for-local-pairing
   {:events [:syncing/preflight-outbound-check]}
@@ -95,7 +101,6 @@
                             (when (sync-utils/valid-connection-string? response)
                               (on-valid-connection-string response)
                               (rf/dispatch [:syncing/update-role constants/local-pairing-role-sender])
-                              (rf/dispatch [:syncing/initiate-pairing-process])
                               (rf/dispatch [:hide-bottom-sheet])))]
     (when-not (and error (string/blank? error))
       (let [key-uid    (get-in db [:profile/profile :key-uid])

